tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -284,7 +284,7 @@ node_modules/chrome-devtools-frontend/front_end/animation/AnimationModel.js(811,
 node_modules/chrome-devtools-frontend/front_end/animation/AnimationModel.js(811,44): error TS2300: Duplicate identifier 'Request'.
 node_modules/chrome-devtools-frontend/front_end/animation/AnimationScreenshotPopover.js(7,11): error TS2339: Property 'AnimationScreenshotPopover' does not exist on type '{ new (effect?: AnimationEffect, timeline?: AnimationTimeline): Animation; prototype: Animation; }'.
 node_modules/chrome-devtools-frontend/front_end/animation/AnimationScreenshotPopover.js(9,23): error TS2304: Cannot find name 'Image'.
-node_modules/chrome-devtools-frontend/front_end/animation/AnimationScreenshotPopover.js(18,39): error TS2345: Argument of type 'new (width?: number, height?: number) => HTMLImageElement' is not assignable to parameter of type 'Node'.
+node_modules/chrome-devtools-frontend/front_end/animation/AnimationScreenshotPopover.js(18,39): error TS2322: Type 'new (width?: number, height?: number) => HTMLImageElement' is not assignable to type 'Node'.
```

Comment:
Did we get rid of the bad-parameter-specific error message? Seems odd.

Reply:
Here's a repro:

```ts
declare var ohno: new () => HTMLImageElement;
declare function ff(t: Node): void;
ff(ohno)
```

I'll file a bug, but I'm not sure how bad this is.

Reply:
Filed #27049 
@weswigham can you decide if this is By Design or not?

Reply:
Hm. When we elaborate we don't bother using the override message right now, since we assume we won't trigger the elaboration right at the top. We could pass is thru with a little depth tracking?